src/org/ods/OdsPipeline.groovy

```diff
@@ -216,18 +216,19 @@ class OdsPipeline implements Serializable {
   }
 
   private void notifyNotGreen() {
-    if (context.notifyNotGreen) {
-      script.emailext(
-        body: '${script.DEFAULT_CONTENT}', mimeType: 'text/html',
-        replyTo: '$script.DEFAULT_REPLYTO', subject: '${script.DEFAULT_SUBJECT}',
-        to: script.emailextrecipients([
-          [$class: 'CulpritsRecipientProvider'],
-          [$class: 'DevelopersRecipientProvider'],
-          [$class: 'RequesterRecipientProvider'],
-          [$class: 'UpstreamComitterRecipientProvider']
-         ])
-      )
-    }
+    String subject =  "Build $context.componentId on project $context.projectId  failed!"
+    String body = "<p>$subject</p> <p>URL : <a href=\"$context.buildUrl\">$context.buildUrl</a></p> "
+
+    script.emailext(
+            body: body, mimeType: 'text/html',
+            replyTo: '$script.DEFAULT_REPLYTO', subject: subject,
+            to: script.emailextrecipients([
+                    [$class: 'CulpritsRecipientProvider'],
+                    [$class: 'DevelopersRecipientProvider'],
+                    [$class: 'RequesterRecipientProvider'],
+                    [$class: 'UpstreamComitterRecipientProvider']
```

Comment:
is this ireally correct ... did you check the receipient list?

Reply:
I actualy do not check or modify the receipient list. It is unchanged from the previous version.
Please keep in mind that the reciepients list can also contain email adresses outsite your organisation.
For this reason, I am going to configure the _Allowed Domains_ property to restrict the email adresses. (out of scope for this pr)